support/module_template.md

```diff
@@ -0,0 +1,13 @@
+# async.<%= name %>
```

Comment:
I don't think so.  It's only there if we were to publish each function as its own package. (same thing with the Gulpfile)